src/proteomes/components/AccessionView.tsx

```diff
@@ -6,34 +6,21 @@ import EntryTypeIcon from '../../shared/components/entry/EntryTypeIcon';
 import { getEntryPath } from '../../app/config/urls';
 
 import { Namespace } from '../../shared/types/namespaces';
-import {
-  ProteomesAPIModel,
-  ProteomeType,
-} from '../adapters/proteomesConverter';
-import { EntryType } from '../../uniprotkb/adapters/uniProtkbConverter';
+import { ProteomesAPIModel } from '../adapters/proteomesConverter';
 
 import './styles/accession-view.scss';
 
-const iconProteomeTypes: ProteomeType[] = [
-  'Reference proteome',
-  'Reference and representative proteome',
-  'Representative proteome',
-];
-
 const AccessionView: FC<Pick<ProteomesAPIModel, 'id' | 'proteomeType'>> = ({
   id,
   proteomeType,
 }) => {
   const link = <Link to={getEntryPath(Namespace.proteomes, id)}>{id}</Link>;
 
-  const showIcon = iconProteomeTypes.includes(proteomeType);
-  return showIcon ? (
+  return (
     <div className="accession-view">
```

Comment:
Realising that this should probably be applied to all the columns that contain the key ID of the table.

Could that be put into a result page stylesheet in shared folder? And the corresponding class be added on all the corresponding column renderers for all the namespaces?

Reply:
Are you talking about the `nowrap`? I think most (all?) ids don't have spaces in them so was questioning the validity of the existence of this component...

Reply:
Ok - I guess I could move the whole component to shared, and change props to be `namespace: Namespace` and `id: string`?

Reply:
Not sure how you'd want to handle icons.
I thought just reusing the class everywhere needed would be easier, but up to you

Reply:
It would be good to have it added on the column renderers for the ID for all the namespaces then.

Reply:
All the supporting data

Reply:
Just the supporting data if we keep the table around for now

Reply:
Ah yes I guess I have to merge the sprint branch into my branch first

Reply:
I got a bit confused about which branch was which so accidentally pushed to https://github.com/ebi-uniprot/uniprot-website/pull/304
I guess there's no point in merging `2021_03-proteomes_card` into `2021_03` anymore and I should just merge https://github.com/ebi-uniprot/uniprot-website/pull/304 ??

Reply:
Yep, not sure what is going where now 😅 . Feel free to close both and get the resulting code on the sprint branch somehow